tests/e2e.test.ts

```diff
@@ -54,16 +54,6 @@ describe('E2E tests', function () {
     if (SERVICE) {
       sleep(10000);
```

Comment:
I believe this is missing an `await` if you want this to actually sleep here....